src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/util/BaseReferralFactory.kt

```diff
@@ -87,6 +90,8 @@ open class BaseReferralFactory(em: TestEntityManager? = null) : EntityFactory(em
         additionalNeedsInformation = additionalNeedsInformation,
         needsInterpreter = needsInterpreter,
         interpreterLanguage = interpreterLanguage,
+        completionDeadline = completionDeadline,
+        maximumEnforceableDays = maximumEnforceableDays,
```

Comment:
Can these be completely removed from `Referral`? They are coming from `referral_details`, right?

Reply:
Yes you are right, I can remove that from referrals and draft referrals